schema.sql

```diff
@@ -10,29 +10,47 @@
 -- );
 -- ALTER TABLE animals ADD COLUMN species VARCHAR(255);
 
-CREATE TABLE owners (
-    id SERIAL PRIMARY KEY,
-    full_name VARCHAR(255) NOT NULL,
-    age INTEGER NOT NULL
-);
+-- CREATE TABLE owners (
+--     id SERIAL PRIMARY KEY,
+--     full_name VARCHAR(255) NOT NULL,
+--     age INTEGER NOT NULL
+-- );
 
-CREATE TABLE species (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255) NOT NULL
-);
+-- CREATE TABLE species (
+--     id SERIAL PRIMARY KEY,
+--     name VARCHAR(255) NOT NULL
+-- );
```

Comment:
- [ ] Please remove unused code so that your code base is clean and uncluttered.